core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-list/dot-categories-list.component.html

```diff
@@ -129,7 +137,18 @@
                                 </dot-action-menu-button>
                             </ng-container>
                         </ng-template>
-                        <ng-template #third>
+                        <ng-template #CategoryName>
+                            <ng-container *ngIf="col.fieldName === 'categoryName'; else last">
+                                <button
+                                    class="link-button p-button-link"
+                                    [label]="category.categoryName"
+                                    (click)="addBreadCrumb(category)"
+                                    pButton
+                                    type="button"
+                                ></button>
+                            </ng-container>
+                        </ng-template>
+                        <ng-template #last>
```

Comment:
As per the requirement in the issue I sent you is the whole row that needs to be clicked, the primeng component allows you to do that:

> When the user clicks in a category row, if have children is should goes into the listing of the children.

Reply:
updated